app/src/main/res/values/strings.xml

```diff
@@ -453,6 +453,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="invalidate_confirmation">Invalidate</string>
     <string name="invalidation_dialog_message">Invalidating the cache causes the quests to be updated next time they are downloaded. The quest cache is invalidated automatically after one week and immediately when a quest you solved turns out to be already answered by someone else.</string>
     <string name="quest_openingHours_emptyAnswer">You need to describe what is on the sign. If there is no sign, you should ask for the opening hours inside.</string>
+    <string name="quest_openingHours_no_sign">Opening hours are unknown</string>
```

Comment:
Actually, it should say "No opening hours sign"

Reply:
I wanted to avoid discouraging people from asking about opening hours. People should use this option only when there is no opening hours sign and there are unable/unwilling to collect it by other means (usually asking worker inside).

Reply:
But this is exactly what the `opening_hours:signed` tag is about.

Reply:
I agree but I think that there are two situations

- opening hours are signed (tag `opening_hours`)
- opening hours are not signed, one is willing and able to collect them anyway (maybe shop is without customers and one is able to ask shopkeeper, maybe one has a local knowledge, maybe...) -> tagging `opening_hours` and tagging `opening_hours:signed`=no is correct but tagging `opening_hours` is preferable
- opening hours are unsigned, one is unable or unwilling to collect the opening hours data -> tag `opening_hours:signed`

For the second case I wanted to avoid tagging `opening_hours:signed` but I may change that if you are sure.

Reply:
@westnordost 

> I may change that if you are sure

ping?

Reply:
Yes, please do.
For the second case, users are slightly encouraged to collect them anyway because the not-signed answer is hidden in the other answers submenu

Reply:
changed